test/e2e_node/README.md

```diff
@@ -0,0 +1,109 @@
+# Node End-To-End tests
+
+Node e2e tests start kubelet and minimal supporting infrastructure to validate the kubelet on a host.
+Tests can be run either locally, against a remote host or against a GCE image.
+
+*Note: Linux only. Mac and Windows unsupported.*
+
+## Running tests locally
+
+etcd must be installed and on the PATH to run the node e2e tests.  To verify etcd is installed: `which etcd`.
+You can find instructions for installing etcd (here)[https://github.com/coreos/etcd/releases].
+
+Run the tests locally: `make test_e2e_node`
+
+Running the node e2e tests locally will build the kubernetes go source files and then start the
+kubelet, kube-apiserver, and etc binaries on localhost before executing the ginkgo tests under
+test/e2e_node against the local instance.
+
+## Running tests against a remote host
+
+The node e2e tests can be run against one or more remote hosts using one of
+* `test/e2e_node/jenkins/e2e-node-jenkins.sh` (gce only)
+* `test/e2e_node/runner/run_e2e.go` (requires ssh)
+* `./e2e_node.test` (requires host access w/ sudo)
+
+*Requires password-less sudo access (over ssh) on the remote host to run the kubelet and kube-apiserver.*
+
+### Configuring a new remote host for testing
+
+A minimal environment is required before tests can be run on a remote host.  Most of this can be setup for you using:
+
+`curl -L https://goo.gl/kxDoYY | sudo bash`
+
+The script executes the following actions if needed:
+* install etcd
+* install docker
+* install lxc and update grub commandline
+* enable tty-less sudo access
+
+And has been tested using the following gce base images:
+* containervm
+* ubuntu trusty
+
+### Running the tests
+
+To run the tests against the remote host you will need to ensure that you can ssh (password-less) into the machine
+using the hostname and have password-less sudo access.
+
+1. If running against a host on gce
+  * Copy test/e2e_node/jenkins/template.properties and fill in GCE_HOSTS
+  * Make sure host names are resolvable to ssh `ssh <host>`.
+    * If needed, run `gcloud compute config-ssh` to add gce hostnames to your .ssh/config so they are resolvable.
+  * Run `test/e2e_node/jenkins/e2e-node-jenkins.sh <path to properties file>`
+  
+2. If running against non-gce host
+  * *Requires password-less ssh and sudo access*
+  * Make sure you can ssh into the host and run sudo `ssh <hostname> -- sudo ls`
+    * If ssh flags are required (e.g. `-i`), they can be passed to the tests using `--ssh-options`
+  * `godep go run test/e2e_node/runner/run_e2e.go --logtostderr --hosts <comma separated hosts>`
+  
+3. Manually copying the test tar.gz and running the tests
+  * Build the tar.gz `godep go run test/e2e_node/runner/run_e2e.go --logtostderr --build-only`
+  * Copy `e2e_node_test.tar.gz` to the remote host
+  * Extract the archive on the remote host `tar -xzvf e2e_node_test.tar.gz`
+  * Run the tests `./e2e_node.test --logtostderr --v 2 --build-services=false --node-name=<hostname>`
```

Comment:
Is this a test binary? If so I think the `--v` flag gets overwritten to the test log level (is that what you intended here?). If you want to set the glog level, you need `--vmodule=*=2`